wandb/integration/tensorboard/monkeypatch.py

```diff
@@ -163,5 +173,8 @@ def __init__(self, logdir, *args, **kwargs):
     wandb.patched["tensorboard"].append([module, "EventFileWriter"])
 
 
-def _notify_tensorboard_logdir(logdir, save=None, root_logdir=None):
-    wandb.run._tensorboard_callback(logdir, save=save, root_logdir=root_logdir)
+def _notify_tensorboard_logdir(
+    logdir: str, save: bool = True, root_logdir: str = ""
+) -> None:
+    if wandb.run is not None:
```

Comment:
happens if `wandb.run` is `None` should we just continue?

Reply:
I thought that it only happens during `wandb.init`, but then remembered that I've seen code where people call it before init'ing, perhaps we've been implicitly relying on late binding (the relevant call lives in this `TBXEventFileWriter` class) Let me try and raise an error like everywhere else, let's see what the CI has to say about it...